packages/vscode-preview/src/main.tsx

```diff
@@ -21,11 +21,13 @@ import { theme } from './theme'
 
 const root = document.getElementById('root') as HTMLDivElement
 const scheme = document.body.classList.contains('dark') ? 'dark' : 'light'
+const nonce = root.getAttribute('nonce') || undefined
+
 // const reactRoot = ReactDOM.createRoot(root)
 ReactDOM.createRoot(root).render(
   <MantineProvider theme={theme} forceColorScheme={scheme}>
     <LikeC4Context>
-      <App />
+      <App nonce={nonce} />
```

Comment:
I just checked, `MantineProvider` has `getStyleNonce` property
And we can get it with `useMantineStyleNonce` hook

Reply:
Good point! Thanks!